classes/Kohana/Pagination.php

```diff
@@ -109,7 +109,7 @@ public function config_group($group = 'default')
         $config['group'] = (string) $group;
 
         // Recursively load requested config groups
-        while (isset($config['group']) AND isset($config_file->$config['group']))
+        while (isset($config['group']) AND isset($config_file->{$config['group']}))
```

Comment:
@kanryu hey, thanks
is that back compatible with php5.6?

Reply:
@seyfer What happened to the rest?